### DISCUSSION TITLE
Unable to create an ID

### DISCUSSION THREAD

**Discussion by captivedungeon** [community/community#146397] [GitHub Education]
### Select Topic Area

General

### GitHub Feature Area

Discussions

### Body

I recently got my edu mail from college and tried to create a new github account to avail student developer pack (if i get verified) but the problem is it shows that the mail id that i received from my college today has already been used to create a github account i tried resetting the password even that is not happening im not really sure what is happening here

**Comment by Stansbury-C** [comment]
I had this issue a few months ago. I was successful in getting the Student Pack then, two months later I was asked to verify and couldn't because my ID was already in use. Quick email but long wait for the student team and they will sort it for you

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @captivedungeon ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you, the Education category. Good luck!
